canonical/maas-ui
# refactor(tags): use new API for deployed machines table

## Done

- Update the table for displaying deployed machines for a tag to use the new API.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine list and tick some deployed machines.
- Use the take action menu to open the tag form.
- type in a new tag name and click the create tag button to open the modal.
- type something into the kernel options field and click create.
- Save the tag form.
- Go to the tags tab and click the delete icon next to the tag you just created.
- Inside the kernel options warning click "Show the deployed machines".
- The table should appear with the machines you just tagged.

## Fixes

Fixes: canonical/app-tribe#1120.
